script/Contracts.sol

```diff
@@ -46,4 +46,8 @@ library MainnetContracts {
     address public constant FRX_ETH_WETH_DUAL_ORACLE = 0x350a9841956D8B0212EAdF5E14a449CA85FAE1C0;
 
     address public constant WBTC = 0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599;
+
+    address public constant CURVE_LP_YNETH_YNLSDE_POOL = 0x1f59cC10c6360DA918B0235c98E58008452816EB;
```

Comment:
Hey could we please remove these extra changes (i think they got moved over from the tokenized strategy branch)

script/Contracts.sol
test/mainnet/helpers/SetupWithdrawer.sol
test/unit/helpers/Etches.sol